open-metadata-resources/open-metadata-deployment/charts/vdc/templates/ibm-igc.yaml

```diff
@@ -0,0 +1,171 @@
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/name: {{ include "myapp.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: ibm-igc-service
+    spec:
+{{ if .Values.ibmigc.internal.enabled }}
+# If we are using a cluster-internal IGC, stand up both the IGC container and the proxy, embedding the Ansible
+# playbooks for loading sample metadata at runtime within the IGC container
+      hostname: infosvr
+      volumes:
+        - name: samples-volume
+          emptyDir: {}
+      imagePullSecrets:
+        - name: regcred
+      initContainers:
+        - name: init-git
+          image: "{{ .Values.image.configure.repository }}:{{ .Values.image.configure.tag }}"
+          command:
+            - "/bin/bash"
+            - "-c"
+            - >
+                cd /samples && git init egeria &&
+                cd egeria &&
+                git remote add origin https://github.com/odpi/egeria.git &&
+                git config core.sparsecheckout true &&
+                echo 'open-metadata-resources/open-metadata-deployment/sample-data/*' >> .git/info/sparse-checkout &&
+                git pull --depth=1 origin master
+          volumeMounts:
+            - mountPath: /samples
+              name: samples-volume
+      containers:
+        - name: proxy
+          image: "{{ .Values.image.egeria.repository }}:{{ .Values.image.egeria.tag }}"
+          imagePullPolicy: {{ .Values.image.egeria.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: {{ include "myapp.fullname" . }}-configmap
+          env:
+            - name: "logging.level.org.odpi.openmetadata.adapters.repositoryservices.igc"
+              value: "DEBUG"
+          ports:
+            - containerPort: 8080
+          readinessProbe:
+            tcpSocket:
+              port: 8080
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            failureThreshold: 6
+          resources: {}
+        - name: ibm-igc
+          image: "{{ .Values.image.ibmigc.repository }}:{{ .Values.image.ibmigc.tag }}"
+          imagePullPolicy: {{ .Values.image.ibmigc.pullPolicy }}
+          ports:
+            - containerPort: 9446
+            - containerPort: 52181
+            - containerPort: 59092
+          livenessProbe:
+            httpGet:
+              path: /ibm/iis/launchpad
+              port: 9446
+              scheme: "HTTPS"
+            initialDelaySeconds: 180
+            periodSeconds: 20
+            failureThreshold: 12
+          readinessProbe:
+            httpGet:
+              path: /ibm/iis/launchpad
+              port: 9446
+              scheme: "HTTPS"
+            initialDelaySeconds: 180
+            periodSeconds: 20
+            timeoutSeconds: 3
+            failureThreshold: 3
+          resources: {}
+          securityContext:
+            capabilities:
+              add: ["IPC_OWNER"]
```

Comment:
A little uncomfortable, but given DB2 needs it may be difficult to address. Where this isn't permitted external IGC can be used anyway. I will experiment (after merge) to see how this can be done with ibm cloud (we should add notes to the readme with anything we find)

Reply:
Precisely: for external IGC this is all skipped, for container-based we don't really have a choice (DB2 won't work, and we require DB2 to run IGC...)